ui/src/app/cohort-search/tree-node/tree-node.component.tsx

```diff
@@ -286,6 +287,11 @@ export class TreeNode extends React.Component<TreeNodeProps, TreeNodeState> {
     }
   }
 
+  get isCOPESurvey() {
+    const {node: {domainId, hasAttributes, id}} = this.props;
+    return !hasAttributes && id === COPE_SURVEY_ID && domainId === DomainType.SURVEY.toString() ;
```

Comment:
We can simplify this condition if we use `subtype === CriteriaType.SURVEY.toString()` instead of checking `domainId` (only the top level nodes have this subtype), that way we don't need `!hasAttributes`.

Also, `id` is the criteria table id that gets generated by Jon's scripts and will likely change eventually. Better to use `conceptId` or do some type of string match on the node name.

Reply:
thanks! i'll use the combination of conceptId and string match as i thought the concept ID was going to be same (? will verify with Jon)

Reply:
Just verified the concept Id will remain the same, hence adding checks for both concept Id and name